.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,101 @@
+name: Publish release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish-release:
+    name: Publish Release
+    runs-on: ubuntu-latest
```

Comment:
Shall we stick to a specific version? Recently we ran in to a weird build failure which turned out to be due to `ubuntu-latest` pulling in 20.04 (it used pull 18).

Reply:
In which repo it happened? is it during release? 

Apparently most of repos are using `ubuntu-latest` including [ballerina-lang](https://github.com/ballerina-platform/ballerina-lang/blob/master/.github/workflows/publish_release.yml#L8)

Reply:
+1

Reply:
Happened for website repo, with some stuffs related to python and aws cli tools

Reply:
In that case shall we leave `ubuntu-latest` for these and keep exact version which ever failing ones.
Usually python versions in Ubuntu 18 and 20 are different and our python script may not support the latest. Ideally we should update that script to support latest